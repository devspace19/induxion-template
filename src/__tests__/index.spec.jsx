import React from 'react';
import { shallow } from 'enzyme';

import { Provider } from './../index';

describe('Provider', () => {
  it('test snapshot', () => {
    const provider = shallow(<Provider />);
    expect(provider).toMatchSnapshot();
  });
});
