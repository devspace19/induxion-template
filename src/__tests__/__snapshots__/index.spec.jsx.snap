// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Provider test snapshot 1`] = `ShallowWrapper {}`;
